pom.xml

```diff
@@ -80,10 +80,11 @@ limitations under the License.
     <joda-time.version>2.9.9</joda-time.version>
     <junit.version>4.10</junit.version>
     <kafka.version>2.0.1</kafka.version>
+    <kudu.version>1.10.0</kudu.version>
```

Comment:
nit: mind putting this after kite (and switch hive to alphabetical for good measure?)